Auth0Tests/OAuth2GrantSpec.swift

```diff
@@ -107,7 +107,7 @@ class OAuth2GrantSpec: QuickSpec {
             it("shoud build credentials") {
                 let token = UUID().uuidString
                 let code = UUID().uuidString
-                let values = ["code": code, "id_token": idToken, "nonce": nonce]
```

Comment:
Now that we're only using `code`, we won't have front-channel id tokens anymore.